[release-3.0] 3.0.2 documentation updates (#315) (#340)
suse-edge/suse-edge.github.io: asciidoc/day2/mgmt-cluster.adoc

helm ls -n metal3-system

# Expected output
NAME    NAMESPACE      REVISION  UPDATED                                  STATUS    CHART         APP VERSION
metal3  metal3-system  2         2024-06-17 12:43:06.774802846 +0000 UTC  deployed  metal3-0.7.3  1.16.0
----

== Cluster API upgrade

> The Cluster API (CAPI) controllers on a Metal^3^ management cluster are not currently managed via Helm, this section describes the upgrade process.

Reviewer: Can you please elaborate more about this section? 
We can mention about not managed via Helm as a note.

[NOTE]
====

> This section assumes you have installed `clusterctl` and configured on your system as described in the <<quickstart-metal3, Metal^3^ quickstart>>

Reviewer: This can be a prerequisite.

====

> When upgrading to Edge 3.0.2 from any previous version it will be necessary to upgrade the RKE2 providers:

Reviewer: ```suggestion
When upgrading to Edge 3.0.2 from any previous version it is necessary to upgrade the RKE2 providers.
```

[,bash]
----
clusterctl upgrade apply --bootstrap "rke2:v0.4.1" --control-plane "rke2:v0.4.1"
----

> WARNING: Please ensure the versions selected align with those described in the <<id-release-notes, Release Notes>>, usage of other upstream releases is not supported.

Reviewer: ```suggestion
WARNING: Ensure the versions selected align with those described in the <<id-release-notes, Release Notes>>, usage of other upstream releases are not supported.
```
